sources/main.cpp

```diff
@@ -207,6 +207,27 @@ int main(int argc, char* argv[]) {
     ButtonHandler* buttonHandler = new ButtonHandler(hwFactory->getInterruptHandler(), yioapi);
     qmlRegisterSingletonType<ButtonHandler>("ButtonHandler", 1, 0, "ButtonHandler", &ButtonHandler::getQMLInstance);
 
+    // reset combination was pressed on startup
+    if (buttonHandler->resetButtonsPressed()) {
+        // create marker file
+        QFile file("/firstrun");
+        if (file.open(QIODevice::WriteOnly)) {
+            file.close();
+        }
+
+        // restore default config
+        if (QFile::exists("/boot/config.json")) {
+            QFile::remove("/boot/config.json");
```

Comment:
Wouldn't renaming the current config.json file be the safer option? Gone is gone...

Reply:
Right, better be safe.